.github/actions/setup-gradle-build/action.yml

```diff
@@ -35,6 +35,6 @@ runs:
         printf "org.gradle.vfs.watch=false\n" >> ~/.gradle/gradle.properties
       shell: bash
 
-    - uses: gradle/gradle-build-action@v2
+    - uses: gradle/gradle-build-action@v3
```

Comment:
This should be replaced by `gradle/actions/setup-gradle@v3`, check https://github.com/gradle/gradle-build-action/releases/tag/v3.0.0-rc.1